cognite_toolkit/_cdf.py

```diff
@@ -453,7 +458,7 @@ def auth_verify(
         raise ToolkitValidationError("Failure to verify access rights.")
 
 
-@_app.command("init")
```

Comment:
Your solution works, an alternative would be 
```python
if featureflag.enabled("FF_INTERACTIVE_INIT"):
    from cognite_toolkit._cdf_tk.prototypes.interactive_init import InteractiveInit

    _app.command("init")(InteractiveInit().interactive)
else:
    @_app.command("init")
    def main_init(...)
```

Reply:
Not sure if there is a value to having the old `init` available on `cdf-tk _init`.